RASPI.md

```diff
@@ -0,0 +1,57 @@
+Raspbian Headless installation:
+
+- Download latest raspbian: https://www.raspberrypi.org/downloads/
+- Flash it on SD Card using Etcher: https://www.balena.io/etcher/ (Windows)
+- Eject the card and plug it in again
+- Create new file in boot directory: wpa_supplicant.conf
+- Write to this file:
+country=PL

```

Comment:
Please use code block for this https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Reply:
And apply that to the rest of this document